lib/auth/refreshclient.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var Auth2Client = require('./oauth2client.js');
+var util = require('util');
+
+/**
+ * User Refresh Token credentials.
+ *
+ * @param {string} clientId The authentication client ID.
+ * @param {string} clientSecret The authentication client secret.
+ * @param {string} refreshToken The authentication refresh token.
+ * @constructor
+ */
+function RefreshClient(clientId, clientSecret, refreshToken) {
```

Comment:
This is a very implementation-oriented name. Perhaps UserRefreshClient would still be consistent with other types but add a hint about what you would use it for.